scripts/generate-entry-i10n.js

```diff
@@ -0,0 +1,124 @@
+const { SUPPORTED_LANGUAGES } = require("./../constants.js");
+const { find } = require("lodash");
+const { db, pgp } = require("../api/helpers/db");
+
+const {Translate} = require('@google-cloud/translate').v2;
+const translate = new Translate({projectId: process.env.GOOGLE_PROJECT_ID});
+
+// getThings('case');
+// getThings('method');
+// getThings('organization');
+
+// TEST PURPOSES
+getThings('case', 8);
+getThings('method', 6);
+getThings('organization', 6);
+
+function getThings(type, limit) {
+  db.any(`SELECT * FROM things WHERE type = '${type}' ORDER BY id DESC LIMIT ${limit}`)
```

Comment:
we should only run translations on entries that are not hidden/soft deleted.

Reply:
@paninee did you see this comment?

Reply:
done